helm/hpcc/templates/_helpers.tpl

```diff
@@ -615,3 +615,41 @@ resources:
     memory: "100M"
 {{- end -}}
 
+
+{{/*
+Create placement related settings
+*/}}
+{{- define "hpcc.doPlacement" -}}
+{{- if .placement.nodeSelector }}
+nodeSelector:
+{{ toYaml .placement.nodeSelector | indent 2  }}
+{{- end -}}
+{{- if .placement.tolerations }}
+tolerations:
+{{ toYaml .placement.tolerations }}
+{{- end -}}
+{{- if .placement.affinity }}
+affinity:
+{{ toYaml .placement.affinity }}
+{{- end -}}
+{{- if .placement.schedulerNames -}}
+{{- range $profileName := .placement.schedulerNames }}
+schedulerName: {{ $profileName }}
```

Comment:
is this correct?
It's converting a list, e.g. schedulerNames: ["my-scheduler"], into a series of key values.

Reply:
If there are multiple profiles it will come:
schedulerName: profile1
schedulerName: profile2

Kuberenetes documentation: https://kubernetes.io/docs/reference/scheduling/config/#profiles